Frameworks/Foundation/NSKeyValueCoding.mm

```diff
@@ -256,6 +255,24 @@ static bool tryGetArrayAdapter(id self, const char* key, id* ret) {
     return true;
 }
 
+static bool tryGetSetAdapter(id self, const char* key, id* ret) {
+    std::string countSelectorString = std::string("countOf").append(std::string{ toupper(key[0]) }).append(key + 1);
```

Comment:
for some reason it's not playing well with toupper, so I'm gonna stick with append